app/cdap/components/RecipeDetails/DetailContainer/index.tsx

```diff
@@ -0,0 +1,175 @@
+/*
+ *  Copyright © 2022 Cask Data, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ *  use this file except in compliance with the License. You may obtain a copy of
+ *  the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *  License for the specific language governing permissions and limitations under
+ *  the License.
+ */
+
+import React from 'react';
+import styled from 'styled-components';
+import { Box, Typography } from '@material-ui/core';
+import { grey } from '@material-ui/core/colors';
+import T from 'i18n-react';
+import { dateFormatting } from 'components/RecipeDetails/utils';
+
+interface IRecipeId {
```

Comment:
i think lakshmi already wrote similar interfaces in create recipe PR
plz check with lakshmi and make sure they resemble, especially when they belong to same entity
and update the interface names and declarations accordingly

i need proper explanation on the fixes in this case, plz provide details information

Reply:
Took the reference from lakshmi's code and renamed the types with the name she used.